### DISCUSSION TITLE
Github Desktop VS Git Bash .?

### DISCUSSION THREAD

**Discussion by AzizBenIsmail** [community/community#50797] [Codespaces] [answered]
### Select Topic Area

Question

### Body

**what is the best github desktop or git bash ?**

**Comment by Berkanktk** [comment, marked as answer]
It's really up to your individual needs and preferences. Here is some considerations:

* GitHub Desktop is a tool with a pretty user-friendly interface that makes it easy to work with Git and GitHub repositories. You can do things like cloning, branching, committing, merging, and pushing code, all without having to do too much complicated stuff. Plus, it's pretty handy for working with a team, since it helps with code review and pull requests.
* Git Bash, on the other hand, is a tool that works through a command line interface, which can be more flexible and powerful than using a GUI. If you like working with a terminal and prefer having more control over what you're doing, Git Bash might be more your speed. It's also good for things like advanced Git commands, shell scripting, and automation.

So, which one should you choose? It really depends on your personal preferences and what you need for your particular project. You might even find that it's helpful to use both tools together, depending on what you're working on. In the end, it's all about finding what works best for you!

Personally, I use GitKraken (GUI) as I find it to be easy to manage and navigate around. But if I want more control over a commit, I quickly shift over to the CLI-based git within the terminal. 😄

**Reply by AzizBenIsmail** [reply]
thanks tell me more about GitKraken

**Reply by Berkanktk** [reply]
Well I would say that it is just GitHub Desktop on steroids. Take a look at it for yourself on their [website](https://www.gitkraken.com). 😄

**Reply by AzizBenIsmail** [reply]
Perfect
